src/app/content/components/Page/PageContent.tsx

```diff
@@ -47,6 +47,11 @@ export default styled(MainContent)`
     }
   }
 
+  .highlight {
+    position: relative;
+    padding-top: 0.3rem;
+  }
```

Comment:
these styles make me nervous, i'll need to do some testing once you update the highlighter library to make sure that:

- the padding doesn't move anything on the page
- the position doesn't break math or other elements inside the highlight spans, and doesn't break text wrapping or anything like that